src/utils/daos.c

```diff
@@ -795,6 +795,8 @@ help_hdlr(struct cmd_args_s *ap)
 
 	stream = (ap->ostream != NULL) ? ap->ostream : stdout;
 
+	fprintf(stream, "daos admin command (v%s)\n", DAOS_VERSION);
```

Comment:
one very minor suggestion is to reword the output to "daos command (v%s)\n"
because there is a recently-added command called daos_admin

Reply:
ok, done